data/seeds/03-ingredients.js

```diff
@@ -0,0 +1,25 @@
+
+exports.seed = function (knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('ingredients').truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('ingredients').insert([
+        { Ingredient_name: 'Meat', amount: 1, unit: 'cup', recipes_id: 1 },
```

Comment:
Why capitalize `Ingredient_name`? It's just a key, not anything special like a class or even a pronoun, so this isn't the typical style of camelCasing that's expected!